livre/eq/Channel.cpp

```diff
@@ -234,9 +266,14 @@ struct Channel::Impl
 #endif
     }
 
-    void frameDraw( const eq::uint128_t& )
+    void frameStart( const uint32_t frameCounter )
     {
+        _drawRange = eq::Range::ALL;
```

Comment:
I did not ( I moved to here ). It is related to sort-last rendering, where I am not yet sure about the full algorithm.